pyomo/repn/plugins/gams_writer.py

```diff
@@ -243,7 +243,7 @@ def _write_model(self,
         # how to deal with, plus Suffix if solving
         valid_ctypes = set([
             Block, Constraint, Expression, Objective, Param,
-            Set, RangeSet, Var, Suffix ])
+            Set, RangeSet, Var, Suffix, Connector ])
```

Comment:
Asking as someone who has never used Connectors before, should we check to make sure that any Connectors in the model are deactivated? So as to not appear as though the writer supports active Connectors?

Reply:
Yes, that seems like a good idea.

Reply:
I'm a bit confused now, because I thought that all of the connectors should have been disabled in the model where I ran into this problem. I'll check again tomorrow.

Reply:
I am also confused because line 247 should only be collecting active ctypes in the model, so the writer supposedly already supports any deactivated ctypes

Reply:
The correct solution (in the New World Order) is for Transformations to deactivate non-standard components after they have been transformed "out" of the model.  So, I believe that "option 1" is the correct way to go.

Reply:
When I get a chance, I'll poke at the ConnectorExpander.

Reply:
I agree with John.  So we can close this PR?

Reply:
Sounds reasonable to me.

Reply:
I am officially changing my mind on this PR.  When I wrote the above, I forgot that Connectors are *not* `ActiveComponent`s, and therefore cannot be deactivated.  I do not think that Connectors should be made Active, either (for the same reason that neither Var or Param are Active).  I believe that while deactivating the component is the correct solution when the component is an `ActiveComponent`, in this case, the correct solution should be for the writer to just ignore the Connectors (see PR #280, design discussion #309).